templates/docs/patterns/tiered-list/index.md

```diff
@@ -85,6 +85,20 @@ respectively.
 View example of the tiered list pattern
 </a></div>
 
+## Additional CTA options
+
+In addition to the CTA block placed below the list, you're also able to add CTA
```

Comment:
Nit in wording: maybe we say something like "you may also add", rather than "you're also able to add". This would make the docs a bit more authoritative and make the developer feel a bit more in control.